setup.py

```diff
@@ -1,19 +1,19 @@
-from setuptools import setup
-
-# hack for bootstrapping and include farm_ng.core.package from it's setup.py
-import pathlib
 from pathlib import Path
-import sys
+from setuptools import setup
 
 from setuptools import Command
 from setuptools.command.develop import develop
 from setuptools.command.egg_info import egg_info
 from setuptools.command.install import install
-from importlib.machinery import SourceFileLoader
 
-package_py = pathlib.Path(__file__).parent.resolve() / "py/farm_ng/core/"
+# pylint: disable=no-name-in-module, import-error
+from farm_ng.package.package import (
+    build_package_protos,
+    clean_protos,
+)  # pylint: enable=no-name-in-module, import-error
 
 
+# TOOD: move `Command`` classes to farm-ng-pytsetup
 class BuildProtosCommand(Command):
```

Comment:
Now we could move these commands in to farm-ng-package ?

Reply:
Yes, stage 2